packages/integration-tests/src/syncTestEnvVarType/syncTestEnvVarType.ts

```diff
@@ -0,0 +1,111 @@
+import type { Node } from '@babel/traverse';
+import _traverse from '@babel/traverse';
+import type { TSTypeElement } from '@babel/types';
+import {
+  assertFile,
+  exportNamedDeclaration,
+  identifier,
+  isIdentifier,
+  isTSPropertySignature,
+  isTSTypeLiteral,
+  tsPropertySignature,
+  tsStringKeyword,
+  tsTypeAliasDeclaration,
+  tsTypeAnnotation,
+  tsTypeLiteral,
+} from '@babel/types';
+import fs from 'fs';
+import { parse, print } from 'recast';
+import parser from 'recast/parsers/typescript';
+
+// @ts-expect-error - issue with esm and traverse: https://github.com/babel/babel/issues/13855
+const traverse = _traverse.default as typeof _traverse;
```

Comment:
Does it still work with cjs ? Because when I did this on Canopia, it worked with locally on vitest but broke the cjs

Reply:
As explained in https://github.com/swarmion/swarmion/issues/585#issuecomment-1470895941, the `cjs` version is used when running `cdk ...` commands